Library/Formula/caf.rb

```diff
@@ -0,0 +1,52 @@
+class Caf < Formula
+  homepage "http://actor-framework.org"
+  head "https://github.com/actor-framework/actor-framework.git"
+
+  stable do
+    url "https://github.com/actor-framework/actor-framework/archive/0.12.0.tar.gz"
+    sha256 "294fb00508c166595d479e9af2fae28f2d370ed164cd1de079ae247acc916801"
+  end
+
+  devel do
+    url "https://github.com/actor-framework/actor-framework.git",
+      :branch => "develop"
+    version "0.13-devel"
+  end
+
+  depends_on "cmake" => :build
+
+  option "with-static", "build as static and shared library"
+  option "with-static-only", "build as static library only"
+  option "with-more-warnings", "build with most compiler warnings"
+  option "with-warnings-as-errors", "elevate compiler warnings to errors"
```

Comment:
I killed a few lower-level options in 20c126c. The compile-time log-level selection is still a bit verbose, alas. If options would support an additional value, we could squash all those into a single option, but it looks like this isn't supported. From a user-perspective, the ability to choose a specific log level has big performance implications, so I would prefer to keep that flexibility.